sql/core/src/test/resources/sql-tests/inputs/operators.sql

```diff
@@ -54,19 +54,4 @@ explain select 3 + 1 || 'a' || 4 / 2;
 explain select 1 == 1 OR 'a' || 'b' ==  'ab';
 explain select 'a' || 'c' == 'ac' AND 2 == 3;
 
--- math functions
-select cot(1);
-select cot(null);
-select cot(0);
-select cot(-1);
 
--- ceil and ceiling
-select ceiling(0);
-select ceiling(1);
-select ceil(1234567890123456);
-select ceiling(1234567890123456);
-
--- floor
-select floor(0);
-select floor(1);
-select floor(1234567890123456);
```

Comment:
After rethinking it, maybe we should keep this file untouched in this PR. The reason is we still can check the type promotion logics here. 

Yeah, we  still should add the test cases in `MathExpressionsSuite.scala`